test/plugins/stats.test.js

```diff
@@ -54,4 +54,30 @@ describe('stats', function () {
       return request(server).get('/probot/stats').expect(404)
     })
   })
+
+  describe('it ignores spammy users', () => {
+    beforeEach(async () => {
+      process.env.IGNORED_ACCOUNTS = 'hiimbex,spammyUser'
```

Comment:
Can you have it delete the `process.env.IGNORED_ACCOUNTS` on `afterEach`? Its a tiny nit but just makes for cleaner tests